backend/backends/base.py

```
@@ -102,13 +102,12 @@ def get_torch_device(
-        try:
-            import intel_extension_for_pytorch  # noqa: F401
-
-            if hasattr(torch, "xpu") and torch.xpu.is_available():
-                return "xpu"
-        except ImportError:
-            pass
+        if hasattr(torch, "xpu") and torch.xpu.is_available():
+            try:
+                import intel_extension_for_pytorch  # noqa: F401 -- enhances XPU perf if available
+            except ImportError:
+                pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

(SIM105)